tokens/src/main.rs

```diff
@@ -31,10 +31,10 @@ fn main() -> Result<(), Box<dyn Error>> {
     let client = RpcClient::new(json_rpc_url);
 
     let exit = Arc::new(AtomicBool::default());
-    let _exit = exit.clone();
+    let exit_for_ctrlc_handler = exit.clone();
     // Initialize CTRL-C handler to ensure db changes are written before exit.
     ctrlc::set_handler(move || {
-        _exit.store(true, Ordering::SeqCst);
+        exit_for_ctrlc_handler.store(true, Ordering::SeqCst);
     })
```

Comment:
nit: For me personally, I think the naming `exit_for_ctrlc_handler` could imply that it is separate from the other exit signal?

I think the following is essentially what the original code was intending to do?
```suggestion
    // Initialize CTRL-C handler to ensure db changes are written before exit.
    ctrlc::set_handler({
        let exit = exit.clone();
        move || {
            exit.store(true, Ordering::SeqCst);
        }
    })
```

Reply:
yeah, that confusion is valid concern: https://github.com/solana-labs/solana/pull/31319/commits/9de98b256734642c9e5c2866e28816363b791c48

(btw, seems I can't automatically apply your suggestion on deleted lines...: 

>  ![image](https://user-images.githubusercontent.com/117807/234150053-32ccb637-87ef-421a-8bad-7bc612dd4eb0.png)
)

Reply:
> (btw, seems I can't automatically apply your suggestion on deleted lines...:

you just don't want to give me credit on the commit! 😿